tests/data/requiredTS/requiredTS.yaml

```
@@ -0,0 +1,9 @@
+name: MAGICC
+required_timeseries:
+  - variable: Emissions|CO2
+    region: World
+    years: [2020, 2025, 2050, 2075, 2100]
```

I'd use decadal timesteps for the tests because that is more intuitive for ou user base
```suggestion
    years: [2020, 2030, 2040, 2050]
```